### PR TITLE
update EndeavourOS logo

### PR DESCRIPTION
**Pull request by Un1q32**
top is before, bottom is after

image

real logo:
image

### CODE REVIEW

File: pfetch-extractor/logos.sh [hunk 1]
```diff
@@ -269,13 +269,13 @@ case ${1:-${PF_ASCII:-${distro:-$os}}} in
         ;;
 
     [Ee]ndeavour*)
-        read_ascii 4 <<- EOF
```
[the remark this answers is not in the document]
Comment (Un1q32) [reply pfetch-extractor/logos.sh:272]: Green? the title is cyan
Screenshot 2024-07-25 at 12 58 45 PM

File: pfetch-extractor/logos.sh [hunk 2]
```diff
@@ -269,13 +269,13 @@ case ${1:-${PF_ASCII:-${distro:-$os}}} in
...
-				      ${c1}/${c4}\\
-				    ${c1}/${c4}/  \\${c6}\\
```
[the remark this answers is not in the document]
Comment (Un1q32) [reply pfetch-extractor/logos.sh:274]: Screenshot 2024-07-25 at 1 00 11 PM

I think dark blue is the better choice here

**Comment by Un1q32** [comment]
I don't want to use 256 color mode, since not all terminals support it, and I want to use the same logo for this and https://github.com/Un1q32/pfetch

File: pfetch-extractor/logos.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gobidev) [reply pfetch-extractor/logos.sh:272]: Sorry my bad, I had a colorscheme active that made it look green

File: pfetch-extractor/logos.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (Gobidev) [reply pfetch-extractor/logos.sh:274]: I agree

**Review by Gobidev — APPROVED** [review]

**Comment by Gobidev** [comment]
> I don't want to use 256 color mode, since not all terminals support it, and I want to use the same logo for this and https://github.com/Un1q32/pfetch. I can't support 256 color modes there since I aim for maximum portability and simplicity.

If 256 color is not available, it should choose the nearest supported color, but I see your point. Thank you for the contribution :)

**Pull request merged**